pkg/ldd/ldd_unix.go

```diff
@@ -80,13 +79,16 @@ func runinterp(interp, file string) ([]string, error) {
 		return nil, err
 	}
 	for _, p := range strings.Split(string(o), "\n") {
-		f := strings.Split(p, " ")
```

Comment:
If you turn the core of this loop into a function, writing a test will be very easy, I hope the extra headache of doing that is ok.